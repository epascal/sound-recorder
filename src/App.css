div#root {
  margin: 0.5rem;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: flex-start;
}

main {
  width: 100%;
  justify-content: center;
  display: flex;
  flex-direction: column;
}

header {
  display: flex;
  width: 100%;
  justify-content: center;
}

main div.commands {
  display: flex;
  flex-direction: column;
  justify-content: center;
}

main input.slider {
  width: 150px;
  margin-top: 15px;
  align-self: center;
}
