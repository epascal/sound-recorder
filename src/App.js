import Recorder from './components/Recorder'
import './App.css';


function App() {

  return (
    <>
    <header>
      <h1>Sound Recorder</h1>
    </header>
    <main>
      <Recorder />
    </main>
    </>
  );
}

export default App;
