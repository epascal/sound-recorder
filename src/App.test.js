import { render, screen } from '@testing-library/react';
import App from './App';

test('renders Sound Recorder title', () => {
  render(<App />);
  const linkElement = screen.getByText(/Sound Recorder/i);
  expect(linkElement).toBeInTheDocument();
});
