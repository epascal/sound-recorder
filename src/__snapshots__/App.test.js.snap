// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Sound Recorder title 1`] = `
Array [
  <header>
    <h1>
      Sound Recorder
    </h1>
  </header>,
  <main>
    <button
      className="record-play"
    >
      â€¦
    </button>
  </main>,
]
`;
